src/minigames/minigame.ts

```diff
@@ -43,4 +56,15 @@ export class Minigame {
     // Show the player they've won or lost.
     this.delegate.onMinigameEnd(passed);
   }
+
+  private onTick() {
+    this.elapsedMS += this.ticker.elapsedMS;
+
+    if (this.lifetime === undefined) {
```

Comment:
I think if this is the case perhaps we shouldn't add the listener at all? not a big deal probs